content/scripts/build.js

```diff
@@ -625,7 +646,16 @@ class Builder {
     const t0 = new Date();
 
     for (const { source, localeFolder, folder, files } of self.walkSources()) {
-      if (self.excludeFolder(source, folder, localeFolder, files)) {
+      if (this.specificFiles.size) {
+        if (
+          !files
+            .map((f) => path.join(folder, f))
+            .some((f) => this.specificFiles.has(f))
```

Comment:
Hmm... In Yari (unlike Stumptown) you never have more than 1 document in a folder. So, I *could* have done this somewhere in the constructor (use your imagination for the code):

```
specificFolders = new Set()
for (const file of options.files) {
  checkThatFileExists(file)
  specificFolders.add(path.dirname(file))
}
```
Then, here, the lookup would be faster than comparing an overlap. I could just do:
```
if (this.specificFolders.size) {
  if (!this.specificFolders.has(folder)) {
    continue;
  }
}
```
But that'd break in Stumptown. Hmm...